### DISCUSSION TITLE
C2 Pro Failed to recognize the file format

### DISCUSSION THREAD

**Discussion by Troubadix** [AT0myks/reolink-fw-archive#60] [General]
Hi,

dowloaded the C2 Pro firmware 3.0.0.136 user "python download.py" but cant update the camera
"Failed to recognize the file format"

Tried downloading twice, same error.

File corrupt? Can anyone provide working update file?

**Comment by AT0myks** [comment]
The download.py script is to download all firmwares for all devices. If you only need a specific one for your camera you can click on your device's name in the list and that will show a table for each of its hardware versions. If you see the one that matches your device you can then choose the firmware version you want and click on it to download it.

The only hardware version of the C2 Pro listed here is IPC_51516M5M. Just to make sure, is that the same version you see in your camera's settings? What's the exact name of the file that you used to do the update?
